pheweb/file_utils.py

```diff
@@ -30,6 +30,7 @@ def get_cacheable_file_location(default_relative_dir, basename):
 
 common_filepaths = {
     'phenolist': os.path.join(conf.data_dir, 'pheno-list.json'),
+    'use_phenos': os.path.join(conf.data_dir, 'use_phenos.txt'),
```

Comment:
Would it be good to have this in DB so easy to change on the fly? 
And maybe caching it so that only every x minutes the list is updated from db.

Reply:
Now the list is only read when the server starts. I find it easier to change a text file than db table in this case but if we add other functionality then db imo. Like to keep it as simple as possible, we don't need to fiddle with the phenos that much